### DISCUSSION TITLE
How do I use modpacks and mods in one profile?

### DISCUSSION THREAD

**Discussion by 64-bitman** [gorilla-devs/ferium#383] [Q&A]
Whenever I upgrade my configured mod-packs or mods, they both overwrite existing mods in the mod folder. How can I have my mods and mod-packs use the same mod folder without them overwriting?

**Comment by theRookieCoder** [comment]
It seems you're trying to add your own mods to the modpack, this isn't supported at the moment. There is a tracking issue for this: #94
